RadarSDK/Include/Radar.h

```diff
@@ -554,6 +555,14 @@ typedef void (^_Nonnull RadarLogConversionCompletionHandler)(RadarStatus status,
                      metadata:(NSDictionary *_Nullable)metadata
             completionHandler:(RadarLogConversionCompletionHandler)completionHandler NS_SWIFT_NAME(logConversion(name:revenue:metadata:completionHandler:));
 
+/**
+logOpenedNotificationConversionWithRequest
+ @param request The request associated with the notification
+
+ @see https://radar.com/documentation/api#send-a-custom-event
+ */
++ (void)logOpenedNotificationConversionWithRequest:(UNNotificationRequest *_Nullable)request NS_SWIFT_NAME(logOpenedNotificationConversion(request:));
```

Comment:
We could try to align this a little closer to the other conversion methods: `logConversionWithNotification`.  What do you think @lmeier @jsani-radar

Reply:
Agree, changed.